pyproject.toml

```diff
@@ -18,7 +18,6 @@ pendulum = "^2.1"
 [tool.poetry.dev-dependencies]
 python-dotenv = "0.19.2"
 pytest = "6.2.5"
-tox = "3.25.0"
```

Comment:
Should we be removing this?

Reply:
ahh no that's a typo thanks for catching!